src/main.ts

```diff
@@ -192,6 +196,8 @@ function launch() {
     },
   });
 
+  mainWindow.removeMenu();
```

Comment:
```suggestion
  Menu.setApplicationMenu(null);
```

You can disable the Package Maker menu bar.